drivers/dac/ltc268x/ltc268x.h

```diff
@@ -47,15 +47,19 @@
 /******************************************************************************/
 /******************* Macros and Constants Definitions *************************/
 /******************************************************************************/
-#define LTC268X_DAC_CHANNELS	16
+#define LTC2686_DAC_CHANNELS          8
+#define LTC2688_DAC_CHANNELS          16
+#define LTC268X_DAC_CHANNELS(id)	  (id ? LTC2688_DAC_CHANNELS : LTC2686_DAC_CHANNELS)
```

Comment:
I'm thinking about adding in the future more devices compatible with this driver.

In that case this macro won't be the best approach.

I'd rather create an array storing these values based on the index (which will be the device id defined with `enum ltc268x_device_id`

Reply:
I agree. Will implement this.